src/Microsoft.ComponentDetection.Detectors/gradle/GradleComponentDetector.cs

```diff
@@ -12,16 +13,27 @@ namespace Microsoft.ComponentDetection.Detectors.Gradle;
 
 public class GradleComponentDetector : FileComponentDetector, IComponentDetector
```

Comment:
Please bump the detector version as it is determining different dependencies now that it contains dev dep info.

Reply:
Ah, I overlooked that detectors were versioned, thanks